chia/util/initial-config.yaml

```diff
@@ -643,7 +643,7 @@ solver:
 
   # Node IDs of trusted solver peers, only these can connect by default
   trusted_peers:
-    localhost: 9999
+    0ThisisanexampleNodeID7ff9d60f1c3fa270c213c0ad0cb89c01274634a7c3cb7: Does_not_matter
```

Comment:
do we need this in the `initial-config.yaml` at all? it looks like something that will be added by `chia init` or at least programmatically.

Reply:
so users will know how to edit, this is the convention its like this for the other trusted peer options as well